common_repo_templates/CODE_OF_CONDUCT.md

```diff
@@ -1,80 +1,3 @@
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
-
-Examples of unacceptable behavior by participants include:
-
-* The use of sexualized language or imagery, and unwelcomed sexual attention or
- advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
- address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
- professional setting
-
-## Our responsibilities
-
-Project maintainers are responsible for clarifying the standards of acceptable
-behavior, and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
-
-Project maintainers have the right and responsibility to remove, edit, or
-reject comments, commits, code, wiki edits, issues, and other contributions
-that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
-threatening, offensive, or harmful.
-
-## Scope
-
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
-
-## Enforcement
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at [acm-contact@redhat.com](mailto:acm-contact@redhat.com). All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
-
-Project maintainers who do not follow or enforce the Code of Conduct in good
-faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
-
-## Attribution
-
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html.
-
-[Homepage]: https://www.contributor-covenant.org
-
-For answers to common questions about this Code of Conduct, see
-https://www.contributor-covenant.org/faq.
+The Open Cluster Management project has adopted the CNCF Code of Conduct. Please view (CODE_OF_CONDUCT.md)[https://github.com/open-cluster-management/community/blob/main/CODE_OF_CONDUCT.md] for details.
```

Comment:
Could you see the changes in https://github.com/open-cluster-management/community/pull/33 its been reviewed and modified by the doc team so I trust that wording a bit better.

I will close that PR once you copy the changes over.

Reply:
Oh, shoot. You did mention that PR. I think it makes more sense to revert the changes here, since there was a request for additional feedback there. I'll revert now.